src/__tests__/CountUp.test.js

```diff
@@ -40,6 +40,48 @@ it('clear previous counter when duration changed', done => {
   }, 1200);
 });
 
+it('re-renders when suffix changes', done => {
+  const { container, rerender } = render(
+    <CountUp duration={1} end={1} suffix="second" />,
+  );
+
+  rerender(<CountUp duration={1} end={30} suffix=" seconds" />);
+
+  setTimeout(() => {
+    const span = container.firstChild;
+    expect(span.textContent).toEqual('30 seconds');
+    done();
+  }, 1000);
+});
+
+it('re-renders when the separator changes', done => {
+  const { container, rerender } = render(
+    <CountUp duration={1} end={1} separator="" />,
+  );
+
+  rerender(<CountUp duration={1} end={30} separator=" " />);
```

Comment:
Maybe we should make **end** to 3000 for example, and then check **span.textContent** to equal "3 000". I think it will be more specific

Reply:
Yeah it makes much more sense. I added the update and also increased the `setTimeout` to 1200 since the duration takes already 1 sec.